tests/hf/test_integrals.py

```diff
@@ -274,3 +277,142 @@ def test_gradient(symbols, geometry, alpha, coeff):
 
     assert np.allclose(g_alpha, g_ref_alpha)
     assert np.allclose(g_coeff, g_ref_coeff)
+
+
+@pytest.mark.parametrize(
+    ("i", "j", "ri", "rj", "alpha", "beta", "d"),
+    [
+        (
+            0,
+            1,
+            pnp.array([0.0]),
+            pnp.array([20.0]),
+            pnp.array([3.42525091]),
+            pnp.array([3.42525091]),
+            pnp.array([0.0]),
+        ),
+    ],
+)
+def test_diff2(i, j, ri, rj, alpha, beta, d):
+    r"""Test that _diff2 function returns a correct value."""
+    assert np.allclose(_diff2(i, j, ri, rj, alpha, beta), d)
+
+
+@pytest.mark.parametrize(
+    ("la", "lb", "ra", "rb", "alpha", "beta", "t"),
+    [
+        (
+            (0, 0, 0),
+            (0, 0, 0),
+            pnp.array([0.0, 0.0, 0.0]),
+            pnp.array([20.0, 0.0, 0.0]),
+            pnp.array([3.42525091]),
+            pnp.array([3.42525091]),
+            pnp.array([0.0]),
+        ),
+    ],
+)
+def test_gaussian_kinetic(la, lb, ra, rb, alpha, beta, t):
+    r"""Test that gaussian_kinetic function returns a correct value."""
```

Comment:
Same comment as above

Reply:
Added a description. The functions are far from each other and the integral is zero.